### DISCUSSION TITLE
Cannot Withdraw BTCpay Balance...Assistance Needed

### DISCUSSION THREAD

**Discussion by etk1980** [btcpayserver/btcpayserver#5510] [Technical Support]
I have tried to be as detailed as possible.

I setup a BTCpay store.  When setting up the wallet I used a xPub key, that I own the private keys to.  These private keys are attached to my blockchain.com wallet.  Previously, when I received payments through BTCpay the payments would appear in my blockchain.com wallet, because that is where the xPub key originated.  

The problem:  The BTCpay payments are not appearing in my blockchain.com wallet, but only on the BTCpay wallet page.  

The solution: I either need to be able to send my bitcoin out from BTCpay or access it via my private keys through a wallet, like Electrum.

When I try to send out the bitcoin from the BTCpay wallet it states that I am using the wrong derivation path, but I have tried the following-
•	m/84'/0'/0'
•	m/44'/0'/0'
•	m/44'/0'

I was told that the reason the payments don’t show in my blockchain.com wallet is because of a Gap Limit problem, basically my blockchain.com wallet stopped fetching the payments from BTCpay.  Blockchain.com wallets don’t allow changing the Gap Limit. Two wallets were suggested that allow the Gap Limit to be raised, Wasabi and Electrum.  When I entered my private keys into Wasabi, my balance showed zero and no transaction history.  When I entered my private keys into Electrum (native segwit) my balance is zero, but a transaction history shows up until May 2023.  In my blockchain.com wallet I do currently have a balance and many transactions since May 2023.  In my Electrum wallet I increased the Gap Limit to 500, but still shows zero balance.

Can anyone assist me with sending out my balance from BTCpay or getting balance to load in Electrum?

**Comment by dennisreimann** [comment]
> I was told that the reason the payments don’t show in my blockchain.com wallet is because of a Gap Limit problem

Correct.

> When I try to send out the bitcoin from the BTCpay wallet it states that I am using the wrong derivation path

On the Wallet Settings page of your BTCPay store, you should find the `Account key path`, which is the Derivation path. This would also be used when sending out the funds via BTCPay Server — but you'd still have to sign the transaction with the key from your blockchain.com wallet.

> When I entered my private keys into Wasabi, my balance showed zero and no transaction history. 

This might be, because it is not looking up the correct derivation path.

>  When I entered my private keys into Electrum (native segwit) my balance is zero, but a transaction history shows up until May 2023. 

You can then use Electrum to find out about the correct derivation path, as it seems to have the correct wallet account connected. Also verify that the xpub (or fingerprint) matches the one you see in the BTCPay Server wallet settings Account Key section. 

> Can anyone assist me with sending out my balance from BTCpay or getting balance to load in Electrum?

You should do this in Electrum, as it already contains your private key and you won't have to import it into yet another wallet (BTCPay Sever). I'm not using Electrum, but I assume this must be some kind of glitch as it should show the balance according to the transactions it has. If not, check if there's an option to rescan the wallet with the increased gap limit.

**Reply by etk1980** [reply]
Thanks for the reply.  From what I have read and see, when re-creating the wallet from seed in Electrum it creates a new xPub.   he xPub in electrum is not the same as the one BTCpay.  But the original wallet, blockchain.com does match BTCpay.

**Reply by etk1980** [reply]
I have tried all of the derivation paths given by Electrum and other sources.

**Reply by dennisreimann** [reply]
>  when re-creating the wallet from seed in Electrum it creates a new xPub

Most likely because it is using a different derivation path. [Walletsrecovery.org](https://walletsrecovery.org/) lists `m/44'/0'/n'` as the default blockchain.com derivation scheme.

**Reply by etk1980** [reply]
Yeah I tried that, but the Electrum won't accept that derivation path for any of its wallet options.  Do you happen to know any other wallets that permit choosing the derivation path?
